runtime/startup_aux.c

```diff
@@ -135,6 +135,7 @@ void caml_parse_ocamlrunparam(void)
       case 'v': scanmult (opt, &caml_verb_gc); break;
       case 'w': scanmult (opt, &caml_init_major_window); break;
       case 'W': scanmult (opt, &caml_runtime_warnings); break;
+      case ',': continue;
       }
       while (*opt != '\0'){
         if (*opt++ == ',') break;
```

Comment:
I don't think we need this `while` loop anymore, then?

Reply:
Well, if I understand correctly, the `scanmult` call does not update `opt`, so we still need to skip over the `=...` part read by it before going on to the next item (which is what the while loop is doing).

Reply:
Ah, I had misunderstood indeed. I thought that we were only skipping commas here.